stonesoup/measures.py

```diff
@@ -8,6 +8,58 @@
 from .types.state import State
 
 
+class GenericMeasure(Base):
+    """Measure base type
+
+    A measure provides a means to assess the seperation between two
+     objects item1 and item2.
```

Comment:
Space causes something weird with formatting...
```suggestion
    A measure provides a means to assess the seperation between two
    objects item1 and item2.
```